lib/structures/Guild.js

```diff
@@ -729,6 +729,9 @@ class Guild extends Base {
             return new Permission(Permissions.all);
         } else {
             let permissions = this.roles.get(this.id).permissions.allow;
+            if(permissions & Permissions.all) {
```

Comment:
```suggestion
            if(permissions & Permissions.administrator) {
```
`Permissions.all` checks if literally all permissions are enabled

Reply:
Oops 🙃